lib/reverse_adoc/converters/a.rb

```diff
@@ -13,7 +13,22 @@ def to_coradoc(node, state ={})
 
         id = id&.gsub(/\s/, "")&.gsub(/__+/, "_")
 
-        Coradoc::Document::Inline::Anchor.new(id: id, href: href, title: title, name: name)
+        if /^_Toc\d+$|^_GoBack$/.match id
+          ""
+        elsif !id.nil? && !id.empty?
+          Coradoc::Document::Inline::Anchor.new(id)
+        elsif href.to_s.start_with?('#')
+          href = href.sub(/^#/, "").gsub(/\s/, "").gsub(/__+/, "_")
+          if name.to_s.empty?
+            Coradoc::Document::Inline::CrossReference.new(href)
```

Comment:
A cross reference links an optional custom text to a target. The target can be an anchor in the document, or an anchor in a separate document.

Reply:
Maybe combine this case with the next case into an initialize method that takes parameters.